ios/HyperSdkReact.mm

```diff
@@ -202,9 +202,18 @@ -(void)stopObserving {
 RCT_EXPORT_METHOD(createHyperServices) {
     if (self.hyperInstance == NULL) {
         self.hyperInstance = [HyperServices new];
+        [HyperSdkReact setHyperServicesReference:self.hyperInstance];
     }
 }
 
++ ( HyperServices *)getHyperServices {
```

Comment:
@Subhash703 These functions are not required since the static prop is not scoped to HyperSdkReact. It can be directly accessed by any class present in this file.

Reply:
Got it. Removed these